src/zck_read_header.c

```diff
@@ -172,24 +173,24 @@ int main (int argc, char *argv[]) {
             chk=zck_get_next_chunk(chk)) {
             char *digest = zck_get_chunk_digest(chk);
             if(digest == NULL) {
-                dprintf(STDERR_FILENO, "%s", zck_get_error(zck));
+                LOG_ERROR("%s", zck_get_error(zck));
                 exit(1);
             }
             char *digest_uncompressed = zck_get_chunk_digest_uncompressed(chk);
             if (!digest_uncompressed)
                 digest_uncompressed = "";
 
-	    if (chk == zck_get_first_chunk(zck)) {
-		    bool has_uncompressed = (strlen(digest_uncompressed) > 0);
-		    if (has_uncompressed)
+        if (chk == zck_get_first_chunk(zck)) {
```

Comment:
Why the whitespace changes?

Reply:
I've adjusted it. I think there were tabs (`\t`) instead of spaces before and my editor changed it. Now it's the proper amount of spaces. I can revert it if you want.